src/composables/deepLinkApi.ts

```diff
@@ -0,0 +1,37 @@
+import { ref } from '@vue/composition-api';
+import VueRouter, { Route } from 'vue-router';
+
+export interface UseDeepLinkApiOptions {
+  router: VueRouter
+}
+
+export function useDeepLinkApi({ router }: UseDeepLinkApiOptions) {
+  const route = ref<Route>(router.currentRoute);
```

Comment:
Not 100% sure, but I think this doesn't need to be ref. Just assign the `router.currentRoute`.